en/docs/learn/extensions/managing-workflow-extension/invoking-the-api-manager-from-the-bpel-engine.md

```diff
@@ -21,7 +21,7 @@ Once the workflow configurations are finalized at the BPEL, the call-back URL of
 </tr>
 <tr class="even">
 <td>REST</td>
-<td><a href="https://localhost:9443/store/site/blocks/workflow/workflow-listener/ajax/workflow-listener.jag" class="uri">https://localhost:9443/store/site/blocks/workflow/workflow-listener/ajax/workflow-listener.jag</a></td>
+<td><a href="https://localhost:9443/api/am/publisher/v0.15/workflows/update-workflow-status" class="uri">https://localhost:9443/api/am/publisher/v0.15/workflows/update-workflow-status</a></td>
```

Comment:
Shouldn't this be the admin api?

https://apim.docs.wso2.com/en/3.0.0/develop/product-apis/admin-apis/admin-v0.15/admin-v0.15/#/Workflows%20(Individual)/post_workflows_update_workflow_status

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/3086/commits/3c32dd32dee72cae7966a47b331fefb27b827837